core/templates/dev/head/profile/Profile.js

```diff
@@ -74,6 +74,14 @@ oppia.controller('Profile', [
       $scope.endingExplorationNumber = 6;
       $scope.Math = window.Math;
 
+      $scope.profileIsOfCurrentUser = function() {
```

Comment:
Don't think this needs to be a function -- the current user isn't going to change. Just:

```
$scope.profileIsOfCurrentUser = (
    GLOBLAS.PROFILE_USERNAME === data.profile_username);
```